assets/src/application/ui/forms/formModal/FormModalForm.tsx

```diff
@@ -45,42 +54,52 @@ const FormModalForm: React.FC<FormModalFormProps> = ({
 		}
 	});
 
+	const submitButton: ButtonProps = {
+		disabled: submitting || pristine,
+		htmlType: 'submit',
+		icon: 'save',
+		onClick: (click) => {
+			click.preventDefault();
+			form.submit();
+			onClose(click);
+		},
+	};
+
+	const resetButton: ButtonProps = {
+		disabled: submitting || pristine,
+		htmlType: 'reset',
+		onClick: (click) => {
+			click.preventDefault();
+			setFormReset(true);
+		},
+	};
+
 	return (
-		<form onSubmit={handleSubmit}>
-			<div style={formStyle}>
-				<FormComponent
-					form={form}
-					values={values}
-					submitting={submitting}
-					pristine={pristine}
-					formReset={formReset}
-					{...formProps}
-				/>
-			</div>
-			<div style={btnRowStyle}>
-				<EspressoButton
-					icon={'save'}
-					type={'submit'}
-					btnType={EspressoButtonType.PRIMARY}
-					buttonText={__('Submit')}
-					disabled={submitting || pristine}
-					onClick={(click) => {
-						click.preventDefault();
-						form.submit();
-						onClose(click);
-					}}
-				/>
-				<EspressoButton
-					buttonText={__('Reset')}
-					disabled={submitting || pristine}
-					onClick={(click) => {
-						click.preventDefault();
-						setFormReset(true);
-					}}
-				/>
-			</div>
-			<pre style={dataStyle}>{JSON.stringify(values, null, 2)}</pre>
-		</form>
+		<Modal
+			visible={true}
+			onOk={onClose}
+			style={modalStyle}
+			okText={__('Submit')}
+			okButtonProps={submitButton}
+			cancelText={__('Reset')}
+			cancelButtonProps={resetButton}
+			onCancel={onClose}
+			width={'80%'}
```

Comment:
perhaps we can set this value as default and allow to configure it, if needed

Reply:
If you don't pass it, AntD will force set it to `520px`